src/style/button.js

```diff
@@ -19,3 +19,10 @@ export const DropboxButton = withStyles(() => ({
     },
   },
 }))(Button);
+
+export const HeartButton = withStyles(() => ({
```

Comment:
Please use a less specific name for this new variant. Let's aim at reusability.

Reply:
What name do you suggest? 😄

Reply:
`BordelessButton` or `TransparentButton`.

It's a great addition, I think we will use this a lot 👍🏼